honeybee_energy/schedule/fixedinterval.py

```diff
@@ -0,0 +1,690 @@
+
+        Note that ladybug DataCollections always follow the "Ladybug Tools
+        Interpretation" of date time values as noted in the
+        ScheduleDay.values_at_timestep documentation.
+
+        Args:
+            timestep: An integer for the number of steps per hour at which to make
+                the resulting DataCollection.
+            start_date: An optional ladybug Date object for when to start the
+                DataCollection. Default: 1 Jan on a non-leap year.
+            end_date: An optional ladybug Date object for when to end the
+                DataCollection. Default: 31 Dec on a non-leap year.
+        """
+        a_period = AnalysisPeriod(start_date.month, start_date.day, 0,
+                                  end_date.month, end_date.day, 23, timestep,
+                                  self.is_leap_year)
+        data_type, unit = self._get_lb_data_type_and_unit()
+        header = Header(data_type, unit, a_period, metadata={'schedule': self.name})
+        values = self.values_at_timestep(timestep, start_date, end_date)
+        return HourlyContinuousCollection(header, values)
+
+    @classmethod
+    def from_idf(cls, idf_string, type_idf_string=None):
+        """Create a ScheduleFixedInterval from an EnergyPlus IDF text strings.
+
+        Args:
+            idf_string: A text string fully describing an EnergyPlus
+                Schedule:File.
+            type_idf_string: An optional text string for the ScheduleTypeLimits.
+                If None, the resulting schedule will have no ScheduleTypeLimit.
+        """
+        # process the schedule inputs
+        sch_fields = parse_idf_string(idf_string, 'Schedule:File')
+        schedule_type = ScheduleTypeLimit.from_idf(type_idf_string) if type_idf_string \
+            is not None else None
+        timestep = 60 / int(sch_fields[8]) if sch_fields[8] != '' else 1
+        start_date = Date(1, 1, False) if sch_fields[5] == '8760' else Date(1, 1, True)
+        interpolate = False if sch_fields[7] == 'No' or sch_fields[7] == '' else True
+
+        # load the data from the CSV file referenced in the string
+        assert os.path.isfile(sch_fields[2]), \
+            'CSV Schedule:File "{}" was not found on this system.'.format(sch_fields[2])
+        all_data = csv_to_matrix(sch_fields[2])
+        transposed_data = tuple(zip(*all_data))
+        csv_data = (float(x) for x in
+                    transposed_data[int(sch_fields[3]) - 1][int(sch_fields[4]):])
+
+        return cls(sch_fields[0], csv_data, schedule_type, timestep, start_date,
+                   0, interpolate)
+
+    @classmethod
+    def from_dict(cls, data):
+        """Create a ScheduleFixedInterval from a dictionary.
+
+        Note that the dictionary must be a non-abridged version for this
+        classmethod to work.
+
+        Args:
+            data: ScheduleFixedInterval dictionary following the format below.
+
+        .. code-block:: shell
```

Comment:
I would use javascript in this case with a JSON object with // comments.

Reply:
I'm afraid that I don't follow your comment here.  I use // comments in the dictionary. Where did you see me not use them?

Reply:
I meant a `.. code-block:: json` will probably give you a better highlight than `.. code-block:: shell`.

I usually use `.. code-block:: python` and then use # for comments as they are python dictionaries but JSON will also work.

Reply:
Ah, that makes sense. Ok. Will do.

Reply:
Done!